frontend/src/style/Header.css

```diff
@@ -19,13 +19,8 @@
   height: auto;
 }
 
-.search-bar {
-  padding: 5px 10px;
-  font-size: 16px;
-  border-radius: 4px;
-  border: 1px solid #ccc;
-  width: 300px;
-  margin: 0 20px;
+.bold {
```

Comment:
This should go in global.css perhaps as its a universal utility class?

Reply:
yes, yes it should